my-fund-backend/src/main/java/com/myfund/services/csv/CsvReaderService.java

```diff
@@ -0,0 +1,28 @@
+package com.myfund.services.csv;
+
+import com.myfund.models.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.util.Map;
+
+@Service
+public class CsvReaderService {
+
+    private final Map<String, CsvParser> parserMap;
+
+    @Autowired
+    public CsvReaderService(Map<String, CsvParser> parserMap) {
+        this.parserMap = parserMap;
+    }
+
+    public void processCsv(String bankName, MultipartFile file, User user, Long budgetId) {
```

Comment:
as this class is named CSV... i would name this method as just parse, csv is redundant